.github/workflows/black.yml

```
@@ -0,0 +1,20 @@
+name: black
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  check-black-formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+        cache: 'pip'
+    - run: pip install black==22.3.0
+    - run: black --diff .
+    - run: black --check .
```

Actually 
```suggestion
    - run: black --diff --check .
```
works as well.
plus I guess we want a newline at the end of the file?